MAVProxy/mavproxy.py

```diff
@@ -18,7 +18,10 @@
 import json
 import struct
 
-from imp import reload
+try:
+    reload
+except NameError:
+    from importlib import reload
```

Comment:
```suggestion
try:
    reload
except NameError:
    try:
        from importlib import reload
    except NameError:
        from imp import reload
```

Reply:
Yeah I guess that would cover all cases, 1 for python2, another for python >3.4 and the other for python <=3.4. Although I don't think any distribution is on a python version <=3.4, besides the older Ubuntu LTS's and all python versions <= 3.4 are EOL'd by Python Foundation themselves.

Reply:
Also the second except should be `ImportError` instead of `NameError` since the error would be thrown on trying to import the module. Updated my commit.